### ISSUE TITLE
Refactor wallet prototypes for FE interface development

### ISSUE THREAD

**Issue by shanejearley** [consensusnetworks/casimir#180]
Clean up wallet composables to any and only useful and intuitive methods/values. The goal here is to ensure that we keep the wallet and auth code nice and decoupled from the UI components when we ramp up building everything out in parallel. 

It might be useful to imagine the wallet.ts composable (and all related sub-composables) as an external library built for both auth/account management and on-chain reads and transactions. The structure of the current separation of the sub-composables can be a higher priority in a separate issue (of course unless it's an easy or necessary refactor).

**Comment by shanejearley** [comment]
We might want to prioritize this over the Trezor bug so we can avoid any hint of tech debt once the face-paced interface buildout begins.
